### DISCUSSION TITLE
Post missing from search results and category tab

### DISCUSSION THREAD

**Discussion by amanda-matthes** [community/community#48989] [Discussions] [answered]
We've been using GitHub Discussions for our project for a few months and have a couple of posts in a few categories. 

I was recently looking for a specific post I had made and could not find it by searching for the title. I then tried checking the category I knew it was in and it also didn't show up. However, I was able to find it by going to "View all discussions" and then scrolling back far enough. 

Here is what the post looks like:
![image](https://user-images.githubusercontent.com/25854078/222456890-a2f34837-6f54-40c3-9a12-123888005279.png)

It's clearly in the category "SnapperGPS in the field" but if I click on the category it is not listed:
![image](https://user-images.githubusercontent.com/25854078/222457386-bc1bc765-d500-4d2b-881f-571095c0cb5b.png)

It also does not show up in search results of the title  "SnapperGPS on sea turtles":
![image](https://user-images.githubusercontent.com/25854078/222457963-b323222a-9e79-4248-97b7-efa7edb3ef87.png)

This is the post I am talking about: https://github.com/orgs/SnapperGPS/discussions/4
This is the category it should show up in: https://github.com/orgs/SnapperGPS/discussions/categories/snappergps-in-the-field
The empty search results if I type in the title: https://github.com/orgs/SnapperGPS/discussions?discussions_q=SnapperGPS+on+sea+turtles

The post is basically impossible to find unless you know it exists. 

It doesn't affect all posts and I am not sure what could be different about this one.

**Comment by LindseyB** [comment]
This is definitely a bug thanks for reporting.

**Comment by LindseyB** [comment, marked as answer]
This should be fixed now.
